Source/LibationUiBase/GridView/GridEntry[TStatus].cs

```diff
@@ -154,6 +154,10 @@ private void UserDefinedItem_ItemChanged(object sender, string itemName)
 			if (udi.Book.AudibleProductId != Book.AudibleProductId)
 				return;
 
+			//If UserDefinedItem was changed on a different Book instance (such as when batch liberating via menus),
+			//EntryStatu's Book instance will not have the current DB state.
+			Liberate.Book = udi.Book;
```

Comment:
That's wacky. I bet tracking down this idiosyncrasy was a challenge and a half.

btw -- typo on `EntryStatu's`

Reply:
Thank you for leaving responsible comments. This is perfect -- it's a note to us later about why this unintuitive thing was done and what exactly was wrong without it.

Reply:
Yeah, I looked at that and thought "I'll probably try to remove this later". Lol